examples/integration/BUILD

```diff
@@ -15,16 +15,54 @@ load(
 
 exports_files(["README.md"])
 
-xcodeproj(
-    name = "xcodeproj",
-    associated_extra_files = ASSOCIATED_EXTRA_FILES,
-    build_mode = "xcode",
-    config = CONFIG,
-    extra_files = EXTRA_FILES,
-    project_name = "Integration",
-    scheme_autogeneration_mode = SCHEME_AUTOGENERATION_MODE,
-    schemes = get_xcode_schemes(),
-    tags = ["manual"],
-    top_level_targets = XCODEPROJ_TARGETS,
-    unfocused_targets = UNFOCUSED_TARGETS,
-)
+# Example of xcodeproj cache warming
+#
+# 1. Register a single macOS execution platform in `WORKSPACE`:
+#
+#      register_execution_platforms("@build_bazel_apple_support//platforms:macos_x86_64")
```

Comment:
Would somebody need to register two execution platforms if they had Apple Silicon on CI?

Reply:
No. AFAIK currently you can only have one execution platform for Apple builds. Even if you define more than one execution platform, only the first one will be used. If you don't define anything, the host platform (`@local_config_platform//:host`) will be used, which is host-dependent. The purpose of this is to use a host-independent platform to build.